airflow/models/__init__.py

```diff
@@ -4376,37 +4376,6 @@ def __repr__(self):
         return self.label
 
 
-class KnownEventType(Base):
```

Comment:
Is an `alembic` file may be needed? Given tables `known_event_type` and `known_event` are removed.

Reply:
Leaving out an alembic script was a conscious decision. The table will be kept for existing airflow setups, but won't be created for the new ones.

Reply:
Makes sense to me.